thevoid/server.cpp

```diff
@@ -524,6 +524,32 @@ int base_server::parse_arguments(int argc, char **argv)
 		return -7;
 	}
 
+	if (config.HasMember("log_request_headers")) {
```

Comment:
Is it documented somewhere?
At least full example config, i.e. with all possible options, with comments - it should be enough for starters